provisioner/clusterpy.go

```diff
@@ -496,6 +496,8 @@ func createOrUpdateEtcdStack(
 	logger.Debugf("Uploaded generated files to %s", s3Path)
 	values[s3GeneratedFilesPathValuesKey] = s3Path
 
+	values["instance_type_info"] = instanceTypes
```

Comment:
Keep an eye out for the rendering function on line 491: https://github.com/zalando-incubator/cluster-lifecycle-manager/pull/874/files#diff-b49199d424770fa971be5fcea8c0799afe324a9aa259c7083b2271c383721333R491 Looks like it renders there, too.